README.md

```diff
@@ -1,22 +1,24 @@
-# fun
+# Fun
 
 (have)Fun with Serverless
 
 ![logo.jpg](./logo.jpg)
 
-[fun](https://github.com/aliyun/fun) 是一个用于支持 Serverless 应用部署的工具，能帮助您便捷地管理函数计算、API 网关、日志服务等资源。它通过一个资源配置文件（template.yml），协助您进行开发、构建、部署操作。
+[中文文档](README-zh.md)
 
-如果您想要使用旧版语法，请[参考](./README.old.md)。
+[Fun](https://github.com/aliyun/fun) is a tool for supporting the deployment of Serverless applications and can help you easily manage resources such as Function Compute, API Gateways, Log Serivce, and so on. It assists you with development, build, and deployment operations through a resource configuration file (template.yml).
```

Comment:
Fun is a development tool for serverless applications . It could help you efficiently arrange cloud resources such as Function compute, API Gateway, Log Service and so on. You can use it to develop，build and deploy FC by describing relative resources in a template.yml file.

Reply:
done